baseapp/baseapp.go

```diff
@@ -974,6 +974,7 @@ func (app *BaseApp) runTx(ctx sdk.Context, mode runTxMode, txBytes []byte) (gInf
 			if len(missingAccessOps) != 0 {
 				for op := range missingAccessOps {
 					ctx.Logger().Info((fmt.Sprintf("Antehandler Missing Access Operation:%s ", op.String())))
+					op.EmitValidationFailMetrics()
```

Comment:
oh were we missing the antehandler failed metrics cus we're only emitting it here?

https://github.com/sei-protocol/sei-cosmos/blob/main/baseapp/baseapp.go#L1121

Reply:
+1, unclear when it would/wouldn't emit now

Reply:
yeah, we were only emitting at the end of delivertx (not sure if those dependencies are included when validating so we may have been emitting anyways, let me double check)

Reply:
I dont believe it would have emitted before since it would return earlier upon error

Reply:
Gotcha, might be useful to add a comment but not a blocker